python/lsst/ts/wep/task/estimateZernikesBase.py

```diff
@@ -275,8 +283,8 @@ def run(
                     f"Wavefront algorithm `{wfEst.algo.__class__.__name__}` "
                     "requires pairs of donuts."
                 )
-            zernikes = self.estimateFromIndivStamps(
+            zernikes, zkMeta = self.estimateFromIndivStamps(
                 donutStampsExtra, donutStampsIntra, wfEst, numCores=numCores
             )
 
-        return pipeBase.Struct(zernikes=zernikes)
+        return pipeBase.Struct(zernikes=zernikes, wfEstInfo=zkMeta)
```

Comment:
Same as above. (I think the return signature for this was actually incorrect, and should have already been a struct?)